katas/main/src/pokerhands/adriel/HandKickersEvaluator.cs

```diff
@@ -0,0 +1,30 @@
+namespace PokerHands.HandsKickersEvaluator.Adriel;
+
+using Cards.Adriel;
+using Hands.Adriel;
+
+public class HandKickersEvaluator
+{
+    public static Card[] Pair(Hand hand)
+    {
+        return hand.Cards.GroupBy(c => c.Value)
+            .Where(g => g.Count() == 1)
+            .Select(e => e.First())
+            .OrderBy(c => c.Value)
```

Comment:
Try to avoid one letter variable names as they are not easy to figure out what value they store

Reply:
Done.